src/rules/comment-no-empty/index.js

```diff
@@ -1,10 +1,16 @@
-import { utils } from "stylelint";
+import { rules, utils } from "stylelint";
 import { namespace } from "../../utils";
 
-export const ruleName = namespace("comment-no-empty");
+const coreRuleName = "comment-no-empty";
+
+export const ruleName = namespace(coreRuleName);
 
 export const messages = utils.ruleMessages(ruleName, {
-  rejected: "Unexpected empty comment"
```

Comment:
This message is the same as the core rule's one, so this PR is just a refactoring.